api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 205**
+* [`POST /register`](/api/register-queue): Add `min_deactivated_realm_deletion_days`
```

Comment:
The missing blank line here results in this page being misrendered; it's best to always test documentation pages because this kind of typographical issue happens all the time if you don't.